examples/handling/interface/topoints.py

```diff
@@ -21,10 +21,10 @@
 # %%
 # We convert the raster to points. By default, this returns a vector with columb geometry burned.
 
-pts_rast = rast.to_points()
+pts_rast = rast.to_pointcloud()
 pts_rast
 
 # %%
 # We plot the point vector.
 
-pts_rast.plot(column="b1", cmap="terrain", legend=True)
+pts_rast.ds.plot(column="b1", cmap="terrain", legend=True)
```

Comment:
Why do we need to add `ds`? Isn't `pts_rast` a vector instance? In which case, the method `plot` should work directly...

Reply:
Indeed, changed back to `plot`!